ios/NativeModules/TextileIPFS.m

```diff
@@ -309,3 +309,12 @@ - (NSString *)_getAccessToken:(NSError**)error {
 }
 
 @end
+
+@implementation RCTBridge (TextileIPFS)
+
+- (TextileIPFS *)ipfs
+{
+  return [self moduleForClass:[TextileIPFS class]];
```

Comment:
This is how you return the single instance of the IPFS node API (our `TextileIPFS` native module). `moduleForClass` is basically a service lookup that returns the single instance of any native module based on class type.